pkg/util/utils.go

```diff
@@ -35,14 +35,16 @@ const (
 
 var (
 	String    = common.String
-	basicTags = []string{common.TagScopeNamespace, common.TagScopeNamespaceUID,
+	basicTags = []string{
+		common.TagScopeNamespace, common.TagScopeNamespaceUID,
 		common.TagScopeStaticRouteCRName, common.TagScopeStaticRouteCRUID,
 		common.TagScopeSecurityPolicyCRName, common.TagScopeSecurityPolicyCRUID,
 		common.TagScopeSubnetCRName, common.TagScopeSubnetCRUID,
 		common.TagScopeSubnetPortCRName, common.TagScopeSubnetPortCRUID,
 		common.TagScopeVPCCRName, common.TagScopeVPCCRUID,
 		common.TagScopeIPPoolCRName, common.TagScopeIPPoolCRUID,
-		common.TagScopeSubnetSetCRName, common.TagScopeSubnetSetCRUID}
+		common.TagScopeSubnetSetCRName, common.TagScopeSubnetSetCRUID,
+	}
```

Comment:
I forget to add TagScopeCluster, TagScopeVersion here, could  you help to add them?

Reply:
Done